src/modules/context2d.js

```diff
@@ -1653,22 +1675,27 @@ import { console } from "../libs/console.js";
         this.lineWidth = lineWidth;
         this.lineJoin = lineJoin;
 
+        var topMargin = (k === min ? this.posY + this.margin[0] : this.margin[0]);
```

Comment:
Should be 1 instead of min here as well.
```suggestion
        var topMargin = (k === 1 ? this.posY + this.margin[0] : this.margin[0]);
```